Library/PagedContainer/PagedContainerViewModel.swift

```diff
@@ -0,0 +1,32 @@
+import Combine
+import Foundation
+import UIKit
+
+public class PagedContainerViewModel {
+  // Internal
+  private var childViewControllers: [UIViewController] = []
+
+  // Inputs
+  func viewWillAppear() {
+    if self.selectedIndex.value == nil {
+      self.didSelectPage(atIndex: 0)
```

Comment:
Philosophically, normally we do this sort of thing by chaining signals on init...but is there any reason _not_ to do it like this? 🤔

Reply:
I could re-write this whole thing to be done on init, if anyone has a strong opinion about it. I didn't do it originally because I refactored this view model out of the view controller, so it's a little less FRP in design.